contracts/src/utils.cairo

```diff
@@ -11,6 +11,25 @@ fn get_beast_address(chain_id: felt252) -> ContractAddress {
     }
 }
 
+// TODO: set correct consumable addresses
+fn get_consumable_address(consumable_id: u8) -> ContractAddress {
```

Comment:
add storage vars for these and provide setters for each so that we can set these after the ERC20Consumables are deployed. Only the deployer of the contract should be able to call these and only a single time, meaning the setters should check if these storage vars have already been set and if so, revert.